test/features/authentication_test.rb

```diff
@@ -0,0 +1,20 @@
+require './test/test_helper'
+
+class Authentication < ActionDispatch::IntegrationTest
+  def test_person_can_log_in_from_omniauth
+   person = create(:person)
+   log_in(person)
+   assert page.has_content? "Your Jobs Dashboard"
+  end
+
+ def log_in(person)
+   OmniAuth.config.test_mode = true
+   OmniAuth.config.mock_auth[:github] = OmniAuth::AuthHash.new(
+     'provider' => 'github',
+     'uid' => person.uid,
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.